vision_evaluation/evaluators.py

```diff
@@ -1046,3 +1046,51 @@ def add_predictions(self, predictions, targets):
             gt_mask = np.asarray(gt_mask)
             mean_l1 = np.abs(pred_mask.astype(np.float)-gt_mask.astype(np.float)).mean()
             self._metric_sum += mean_l1
+
+
+class GroupWiseEvaluator(Evaluator):
```

Comment:
pls add it to __init__.py as well

Reply:
resolved. PR updated.

Reply:
Can you check how other evaluator is added in \_\_init\_\_.py?

Reply:
updated